src/meta/app/src/schema/table.rs

```diff
@@ -165,6 +166,12 @@ pub struct TableStatistics {
     pub index_data_bytes: u64,
 }
 
+#[derive(serde::Serialize, serde::Deserialize, Default, Clone, Debug, Eq, PartialEq)]
+#[serde(default)]
+pub struct TableStorageParams {
+    pub storage: StorageParams,
+}
+
```

Comment:
Why is it necessary to create a wrapper type for `StorageParam`?

Is it ok to use `StorageParam` directly?

Reply:
Ok, but I prefer to add a wrapper since `StorageParam` is not only used by table meta.